src/DonorProfiles/Tabs/TabsRegister.php

```diff
@@ -49,11 +49,11 @@ public function hasTab( $id ) {
 	 * @return string
 	 */
 	public function getTab( $id ) {
-		if ( ! isset( $this->tabs[ $id ] ) ) {
-			throw new \InvalidArgumentException( "No migration exists with the ID {$id}" );
+		if ( ! $this->hasTab( $id ) ) {
+			throw new \InvalidArgumentException( "No tab exists with the ID {$id}" );
```

Comment:
There is probably a better exception that we can throw, or throw a custom exception.

`InvalidArgumentException` is thrown "if an argument is not of the expected type."

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5577/commits/fdaa13fd9a0dd47ee65e037b271fa2e7e60caa8c